ev2/event-driven/vis/IPT.cpp

```diff
@@ -104,9 +107,19 @@ bool vIPT::computeForwardReverseMaps(int cam)
     mat_reverse_map[cam] = cv::Mat(size_shared, CV_32FC2);
 
     //mat_reverse_map fill
-    cv::initUndistortRectifyMap(cam_matrix[cam], dist_coeff[cam], rotation[cam],
-                                projection[cam], size_shared, CV_32FC2,
-                                mat_reverse_map[cam], cv::noArray());
+
+    if(distortion_model == "b'radtan'") {
+        cv::initUndistortRectifyMap(cam_matrix[cam], dist_coeff[cam], rotation[cam],
+                                    projection[cam], size_shared, CV_32FC2,
+                                    mat_reverse_map[cam], cv::noArray());
+    } else if (distortion_model == "equidistant") {
```

Comment:
what happens if no distortion model is provided? or if there is a spelling mistake in the distortion model? seg fault?